social_core/backends/okta.py

```diff
@@ -0,0 +1,79 @@
+"""
+Okta OAuth2 and OpenIdConnect:
+    https://python-social-auth.readthedocs.io/en/latest/backends/okta.html
+"""
+import sys
+if sys.version_info[0] < 3:
+    import urlparse as urlparse
+else:
+    import urllib.parse as urlparse
+
+import requests
```

Comment:
requests is not use in this module, is the import necessary?

Reply:
I've reorganised the imports to be more efficient.